home/models.py

```diff
@@ -122,3 +140,16 @@ class Meta:
 
     def __str__(self):
         return f"{self.question}, Tag : {self.tag}"
+
+    # Input: array of strings that represent name of tags and a question.
+    # The function will go through the array and will add them as a pair - question and tag.
+    @classmethod
+    def add_quetion_tags(cls, tags, question_id):
+        question = Question.objects.get(pk=question_id)
+        for tag_name in tags:
+            tag = Tag.objects.get(tag_name=tag_name)
```

Comment:
you are running a separate DB query for each and every tag, think if you can somehow optimize this to reduce the amount of queries.